lib/password.flow

```diff
@@ -5,6 +5,17 @@ export {
 		password : string, 
 		hash : string 
 	) -> bool = FlowPassword.validateHash;
+
+	// < 0: means no expiration
+	// 0: use default expiration time (84600 sec -> 1 day)
+	// > 0: use specified value 
+	native getPasswordValidationStringExp : (exp: int) -> string = FlowPassword.getPasswordValidationString;
+	// get validation string without expiration (create_reset_password_validation_string(null))
+	getPasswordValidationString : () -> string;
+
+	native getValidationStringExpiration : (vs : string) -> int = FlowPassword.getValidationStringExpiration;
+	native checkValidationStringExpirationExp : (vs : string, exp: int) -> bool = FlowPassword.checkValidationStringExpiration;
```

Comment:
It would seem to me like the checkValidationStringExpirationExp does not need to be an native?

Reply:
agree